styles.css

```diff
@@ -0,0 +1,16 @@
+body{
+    background-color: blue;
+    font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+form{
+    background-color: darkblue;
+    color: white;
+}
+div {
+    padding: 0.1cm;
```

Comment:
I highly recommend you use some common units of length in web development like px, rem, and others. This will make your codes be understood easily by other developers!